inmap_test.go

```diff
@@ -580,7 +580,14 @@ func TestConverge(t *testing.T) {
 			t.Logf("%s completed after %d iterations.", convergenceNames[i], iterations)
 		}
 
-		r, err := d.Results(false, true, map[string]string{"PrimPM25": "PrimaryPM25"})
+		o := NewOutputter("", false, map[string]string{"PrimPM25": "PrimaryPM25"}, nil)
+
+		err := o.CheckForDerivatives()
+		if err != nil {
+			t.Error(err)
+		}
+
+		r, err := d.Results(o.AllLayers, o.OutputVariables, o.GetVariables, o.OutputFunctions)
```

Comment:
I think I understand. I just know that Go can be strict about variables that are defined but not used. It appears, though, that including a struct as a parameter is not equivalent to defining each of its fields within the function.